user/views.py

```diff
@@ -42,3 +127,8 @@ def post(self, request, *args, **kwargs):
             form.save()
             return redirect('user:shows')
         return render(request, self.template_name, {'form': form})
+
```

Comment:
there is already a movie_list view so why is this added?

Reply:
only due to naming view are conventionally named as MovieListView end with you i is not necessary though

Reply:
you have made a class based view while the older one is a function based view. They are both the same though so ig you can remove this.